inc/options/class-wpseo-options.php

```diff
@@ -216,6 +216,22 @@ public static function get_option( $option_name ) {
 		return $option;
 	}
 
+	/**
+	 * Retrieve a single field from an option for the SEO plugin.
+	 *
+	 * @param string $option_name The option the key should come from.
+	 * @param string $key         The key it should return.
+	 *
+	 * @return mixed|null Returns value if found, null if not.
+	 */
+	public static function get_option_value( $option_name, $key ) {
```

Comment:
Suggestion: I think it would be nice to have a `$default = null` parameter as well. This allows us to continue a process with having a default value in the situation where the key does not exist.

Reply:
Sounds good, sec.